docs/installation.md

```diff
@@ -110,20 +110,16 @@ To create an elementary OS install drive on macOS you'll need a USB flash drive
 
 <a href="http://www.etcher.io/" class="button suggested-action">Download Etcher</a>
```

Comment:
I think this should be `https://etcher.io/`; the URL now supports HTTPS

Reply:
Agreed